soccer/gameplay/plays/restarts/basic_indirect.py

```diff
@@ -32,19 +80,8 @@ def __init__(self, indirect=None):
             prekick_timeout=9)
         self.add_subbehavior(pass_behavior, 'pass')
 
-        self.add_transition(
-            behavior.Behavior.State.running, behavior.Behavior.State.completed,
-            lambda: self.subbehavior_with_name('pass').is_done_running() and 
-            self.subbehavior_with_name('pass').state != tactics.coordinated_pass.CoordinatedPass.State.timeout,
-            # Keep trying pass until timeout
-            'pass completes')
 
-    @classmethod
-    def score(cls):
-        gs = main.game_state()
-        return 0 if behavior.Behavior.State.running or (
-            gs.is_ready_state() and gs.is_our_indirect_kick()) else float("inf")
-
-    @classmethod
-    def is_restart(cls):
-        return True
+    def calc_move_pt(self):
+        receive_pt = robocup.Point(0, 3 * constants.Field.Length / 4)
+        ball = main.ball().pos
```

Comment:
This may blow up if the ball is invalid, not sure if it will always be valid if this is called though

Reply:
@JNeiger any thoughts on this?

Reply:
Tanner says this should work as intended